### PR TITLE
chore(i18n,docs): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

**Review by raisedadead — APPROVED** [review]

**Review by DanielRosa74 — APPROVED** [review]
LGTM

File: docs/i18n/espanol/curriculum-file-structure.md [hunk 1]
```diff
@@ -103,4 +103,4 @@ Al renombras un solo archivo de desafío, tienes que:
 
 ## La propiedad `dashedName`
 
-La propiedad `dashedName` se usa para generar la ruta URL para el súper bloque, bloque o desafío. Por lo general, estos deben coincidir con los nombres que generaría el asistente `/utils/dasherize.js` para el nombre del archivo.
+The `dashedName` property is used to generate the URL path for the superblock, block, or challenge. These should generally match what the `/utils/slugs.js` helper would output for the file name.
```
Comment (RafaelDavisH) [inline docs/i18n/espanol/curriculum-file-structure.md:106]:
```suggestion
La propiedad `dashedName` se usa para generar la ruta URL para el súper bloque, bloque o desafío. Por lo general, estos deben coincidir con los nombres que generaría el asistente `/utils/slugs.js` para el nombre del archivo.
```

**Review by RafaelDavisH — COMMENTED** [review]
~Suggestion made~. LGTM

**Review by RafaelDavisH — COMMENTED** [review]
👍🏼  LGTM

**Pull request merged**